services/app-api/src/handlers/apollo_gql_express.ts

```diff
@@ -0,0 +1,144 @@
+// PREPARATION FOR APOLLO SERVER 4 MIGRATION
+// This entrypoint sets up Apollo Server using Express, with CORS and body parsing middleware.
+// It is NOT currently used in production. When upgrading to Apollo Server 4, switch to this handler.
+
+// If you see a type error for 'aws-serverless-express', add a declaration file as below:
+// declare module 'aws-serverless-express';
+
+import express from 'express'
+import { ApolloServer } from 'apollo-server-express'
+import cors from 'cors'
+import { createServer, proxy } from 'aws-serverless-express'
+import typeDefs from '../../../app-graphql/src/schema.graphql'
+import { configureResolvers } from '../resolvers'
+import { configurePostgres, configureEmailer } from './configuration'
+import { newJWTLib } from '../jwt'
+import { ldService, offlineLDService } from '../launchDarkly/launchDarkly'
+import * as ld from '@launchdarkly/node-server-sdk'
+import { NewPostgresStore } from '../postgres/postgresStore'
+import {
+    newAWSEmailParameterStore,
+    newLocalEmailParameterStore,
+} from '../parameterStore'
+import {
+    ApolloServerPluginLandingPageDisabled,
+    ApolloServerPluginLandingPageLocalDefault,
+} from 'apollo-server-core'
+import type { S3ClientT } from '../s3'
+import type { APIGatewayProxyEvent, Context as LambdaContext } from 'aws-lambda'
+
+// --- Express app setup ---
+const app = express()
+
+// CORS and body parser middleware (mirroring current config)
+app.use(cors({ origin: true, credentials: true }))
+app.use(express.json({ limit: '10mb' }))
+
+// --- Apollo Server setup ---
+async function buildApolloServer() {
+    // These env vars and config mirror your current handler
+    const stageName = process.env.stage || 'local'
+    const dbURL = process.env.DATABASE_URL || ''
+    const secretsManagerSecret = process.env.SECRETS_MANAGER_SECRET || ''
+    const applicationEndpoint = process.env.APPLICATION_ENDPOINT || ''
+    const emailerMode = process.env.EMAILER_MODE || 'LOCAL'
+    const parameterStoreMode = process.env.PARAMETER_STORE_MODE || 'LOCAL'
+    const ldSDKKey = process.env.LD_SDK_KEY || ''
```

Comment:
The majority of these env vars are required for anything to work, so I think we might want to do some validations that they're set rather than falling back to empty strings. We can probably just copy/paste those over from our current handler, or write a `validateEnvVars` function of some sort if we want to encapsulate it a little better in this refactor.